js16.js

```diff
@@ -0,0 +1,62 @@
+// HOME WORK
+
+// Reverse str
+const reverse = (str) => {
+    let reversedStr = '';
+    for (let i = str.length - 1; i >= 0; i--) {
+        reversedStr += str[i];
+    }
+    return reversedStr;
+};
+const reversedString = reverse('hello');
+console.log(reversedString);
+
+// Count words
+const countWords = (str) => {
+   str = str.trim();
+  const words = str.split(/\s+/);
+  return words.length;
+};
+
+const sentence = "Hi my name is Taras";
+const wordCount = countWords(sentence);
+console.log(wordCount);
+
+// Clear whitespace
+const clearWhiteSpace = (str) => {
+    let stringWithoutSpaces = '';
+    for (let i = 0; i < str.length; i++) {
```

Comment:
You had a great regex to detect whitespace
Try to solve this with replace or replaceAll